### PR TITLE
[ArcResourceBridge] Arc Resource Bridge update apiversion 2022-10-27

### PR DESCRIPTION
**Pull request by lijinpei2008**
## Description
Design: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1300

Swagger: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/resourceconnector/resource-manager/Microsoft.ResourceConnector/stable/2022-10-27/appliances.json

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by lijinpei2008** [comment]
Discussions with the service team: https://msazure.visualstudio.com/One/_workitems/edit/15381613/

**Comment by lijinpei2008** [comment]
After discussion with the service team, the names of some commands were changed
- Get-AzArcResourceBridge
- Get-AzArcResourceBridgeKey              ->  Get-AzArcResourceBridgeCredentials
- Get-AzArcResourceBridgeTelemetryConfig
- Get-AzArcResourceBridgeUpgradeGraph
- Get-AzArcResourceBridgeUserCredential   ->  Get-AzArcResourceBridgeApplianceCredential
- New-AzArcResourceBridge
- Remove-AzArcResourceBridge
- Update-AzArcResourceBridge

**Comment by lijinpei2008** [comment]
User need to provide a real vCenter or vSphere. 
The service team is responsible for building the required environmental validation.

File: src/ArcResourceBridge/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      subject: ApplianceTelemetryConfig
+    set:
+      subject: TelemetryConfig
+  - where:
+      subject-prefix: ArcResourceBridge
+      subject: ApplianceUpgradeGraph
+    set:
+      subject: UpgradeGraph
+  - where:
+      subject-prefix: ArcResourceBridge
```
Comment (VeryEarly) [inline src/ArcResourceBridge/README.md:106]: why not combine directive at line 85 with this one? in the discussion you provided ClusterUserCredential is expected, why you need these two directives?

Comment (lijinpei2008) [reply src/ArcResourceBridge/README.md:106]: This code has been merged, and an omission occurred when the command name was changed several times.

Comment (lijinpei2008) [reply src/ArcResourceBridge/README.md:106]: Mentioned in the email before I hide the command "Get - AzArcResourceBridgeApplianceCredential" and"
The Get - AzArcResourceBridgeCredential "but service team don't agree to my idea.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**